poanetwork/blockscout
# Add pagination and async load to blocks page

Depends on #1154. Closes #1145.

This PR adds async load and simple pagination to the blocks, uncles and reorgs pages.

## Things to test

- Check that the blocks, uncles and reorgs pages show the expected lists.
- Check that the next page button appears when there are more blocks to show.
- Check that the next page button is hidden when the user is on the last page.
- Check that new blocks appear in real time, but only in the first page.
- Check that the placeholders for missing blocks are shown.
